cmd/containerd-shim-runhcs-v1/pod.go

```diff
@@ -228,6 +228,19 @@ func (p *pod) ID() string {
 	return p.id
 }
 
+func (p *pod) getCloneAnnotations(ctx context.Context, s *specs.Spec) (isTemplate bool, templateID string, err error) {
+	templateID = oci.ParseAnnotationsTemplateID(ctx, s)
+	isTemplate = oci.ParseAnnotationsSaveAsTemplate(ctx, s)
+	if templateID != "" && isTemplate {
+		return false, "", errors.Wrapf(errdefs.ErrInvalidArgument, "templateID and save as template flags can not be passed in the same request.")
```

Comment:
don't put periods at the end of error strings, as it doesn't play nice with error wrapping.